packages/server-wallet/src/wallet/store.ts

```diff
@@ -398,7 +398,11 @@ export class Store {
         tx
       );
 
-    await ObjectiveModel.insert(objectiveToBeStored, tx);
+    try {
+      await ObjectiveModel.insert(objectiveToBeStored, tx);
+    } catch (e) {
+      if (e.name !== 'UniqueViolationError') throw e;
```

Comment:
You could also upsert, ie. `ON CONFLICT whatever_the_name_of_the_constraint_is DO NOTHING`, but it's difficult to do that with `knex`.

The current implementation is potentially dangerous: we might later have a _different_ uniqueness constraint that gets violated, and that error _should_ result in throwing `e`. IE. the test `e.name !== 'UniqueViolationError'` can have false negatives.